Last.FM/chain.js

```diff
@@ -0,0 +1,43 @@
+function TailUrl() { 
```

Comment:
It will 1000 instances of TailUrl().
This is standart pattern [https://github.com/shichuan/javascript-patterns/blob/master/object-creation-patterns/chaining.html](url) for object creation.